editor/test/test.html

```diff
@@ -110,6 +110,7 @@ <h2>🔒 Your Privacy is Protected</h2>
                 "test_local_storage_keeper": "./test_local_storage_keeper.js?v=EPOCH",
                 "test_main": "./test_main.js?v=EPOCH",
                 "test_report_data": "./test_report_data.js?v=EPOCH",
+                "test_results": "./test_results.js?v=EPOCH",
```

Comment:
For this PR, let's check that the required elements are always present so we aren't testing for null in `test_results.js`.